code/lib/preview-api/src/modules/store/csf/prepareStory.test.ts

```diff
@@ -15,21 +26,42 @@ vi.mock('@storybook/global', async (importOriginal) => ({
 const id = 'id';
 const name = 'name';
 const title = 'title';
-const render = (args: any) => {};
+const render = () => {};
```

Comment:
Changed this, as there are subtle differences between prepareMeta and prepareStory when the renderer has args, which were not relevant for those tests.